Core/Plugins/Basic/Image404.cs

```diff
@@ -116,7 +116,7 @@ public enum FilterMode
                     !string.Equals(value, "auto", StringComparison.OrdinalIgnoreCase); }),
 
             new Matcher("cropxunits"),
-            new Matcher("cropyyunits"),
+            new Matcher("cropyunits"),
```

Comment:
This is suspicious

Reply:
This is a bugfix, although unlikely to be encountered.

Reply:
As in, your change is the bug fix.